airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -174,6 +186,7 @@ public AirbyteCatalog discover(final JsonNode config) throws Exception {
           .map(MySqlSource::overrideSyncModes)
           .map(MySqlSource::removeIncrementalWithoutPk)
           .map(MySqlSource::setIncrementalToSourceDefined)
+          .map(MySqlSource::setDefaultCursorFieldForCdc)
           .map(MySqlSource::addCdcMetadataColumns)
```

Comment:
We need to update the method `addCdcMetadataColumns` to add the new column as part of the catalog.

Reply:
See L161.